src/client/pages/farm/parcels.vue

```diff
@@ -7,23 +7,28 @@
     >
       Percelen selecteren <md-icon>arrow_left_alt</md-icon>
     </md-button>
-    <parcels-table :parcels="features" />
+    <no-ssr>
+      <parcels-table :parcels="features" />
```

Comment:
should we call them `parcels` everywhere instead of `features`?

Reply:
i can do this in a seperate PR